workshops/test/test_sponsorship.py

```diff
@@ -72,7 +72,7 @@ def setUp(self):
         self.sponsorship = Sponsorship.objects.create(
             organization=self.org_alpha,
             event=self.event,
-            amount=100.00,
```

Comment:
`100.00` is not a valid choice variable to the `amount` field when we run the tests with PyData deployed.